Sources/ComposableArchitecture/SharedState/PersistenceKey/AppStorageKey.swift

```diff
@@ -288,7 +288,8 @@ extension AppStorageKey: PersistenceKey {
   }
 
   public func subscribe(
-    initialValue: Value?, didSet: @escaping (_ newValue: Value?) -> Void
+    initialValue: Value?,
+    didSet: @Sendable @escaping (_ newValue: Value?) -> Void
```

Comment:
Making `didSet` sendable since it will typically be used in concurrent situations.